docs/guides/faq.html.md

```diff
@@ -0,0 +1,28 @@
+---
+title: FAQ
+subtitle: Plugin creation
+layout: guide
+order: 0
+---
+
+This ensures that Danger only runs when you have the environment variables set up to run.
+
+### Can Danger comment inline on an PR?
+
+Not yet, but there is a lot of discussion on [danger-js#77][77]
+
+### I want to help influence Danger's direction
+
+We'd recommend first becoming acquainted with the [VISION.md][] inside Danger, this is the long-term plan. Then there are two ways to start contributing today:
+
+ * Opinions are extra welcome on issues marked as [Open For Discussion][open].
+ 
+ * Well defined work items like features or fixes are marked as [You Can Do This][you-can-do-this].
+
+We keep comments in the public domain, there is a Slack, but it's very rarely used. If you're interested in joining, you can DM [@orta][].
```

Comment:
I don't think `[@orta]` links to anything right now, at least according to GitHub's rich diff:

![image](https://cloud.githubusercontent.com/assets/2344137/25010297/b5e649de-2037-11e7-8588-a84f35a7e099.png)